src/app/api/rooms/route.js

```diff
@@ -138,33 +138,18 @@ export const dynamic = "force-dynamic";
 export async function POST(request, context) {
   const roomData = await request.json();
   const profile = await getProfile();
-
-  try {
-    if (roomData.buildingId) {
-      const room = await prisma.room.create({
-        data: {
-          name: roomData.name,
-          members: {
-            connect: { id: profile.id },
-          },
-          building: {
-            connect: { id: roomData.buildingId },
-          },
-          shoppingLists: {
-            create: {},
-          },
-          choreLists: {
-            create: {},
-          },
-        },
-      });
+  const managerProfile = await prisma.profile.findFirst({
```

Comment:
Why are you looking for the first manager available here? I'm not sure I understand the flow.

Reply:
i'll fix it to filter by roomid or buildingid where provided